src/components/ui/Input/Input.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * This file is based on the Input component from shadcn and customized for our needs.
+ *
+ * See the official docs for more info:
+ * shadcn/ui: https://ui.shadcn.com/docs/components/input
+ */
+
+import { cn } from '@/lib/tailwind/utils';
+
+import { cva, VariantProps } from 'class-variance-authority';
+import { forwardRef, InputHTMLAttributes } from 'react';
+
+const inputVariants = cva(
+  'flex w-full h-full text-base focus:outline-none disabled:cursor-not-allowed',
+  {
+    variants: {
+      variant: {
+        // naming of each variant should be aligned with the design team
+        rounded: 'rounded-full bg-white border border-gray px-6 py-4 placeholder:text-gray',
+        muted: 'border-none placeholder:text-gray',
```

Comment:
muted is basically minimum design without any padding. Used in SearchInput component

Reply:
It's already used in SearchInput, and except for that, I'm actually not sure where we're gonna use it.
Here's why I created "muted" variant. Please read through them and tell me your point of view 👍 

1.  Even if it will be only used in SearchInput, leaving "muted" variant doesn't have any negative effect to this Input component
2. It might be better than create "muted" input component separately or define it in inside of "SearchInput" component since it shares base styling classes.

Reply:
Oh, I'm sorry. I forgot to delete the comment. I wrote the comment before noticing it was used in the `SearchInput`.
I think you can leave it as a variant.

What I noticed it that `muted` should have a transparent background. When I tried it, it had a white background.

Reply:
Nono, it was a good call out to consider tho.
I'll leave the variant as it is, and yeah,  transparent background would be definetely better! I'll set it!